group/short.go

```diff
@@ -254,11 +268,22 @@ func (e *wElt) UnmarshalBinary(b []byte) error {
 		}
 		e.x, e.y = x, y
 	case l == 1+2*byteLen && b[0] == 0x04: // uncompressed
-		x, y := elliptic.Unmarshal(e.wG.c, b)
-		if x == nil {
+		var err error
+		switch bitSize {
```

Comment:
Maybe better match on `e.wG.c` if we ever want to add Curve25519 or some other curve with clashing bitlen.